src/components/RedirectTile/RedirectTile.tsx

```diff
@@ -0,0 +1,68 @@
+import { ComponentPropsWithoutRef, ReactNode } from 'react';
+import { openInNewTab } from '../../util/utils';
+
+export interface RedirectTileProps extends ComponentPropsWithoutRef<'div'> {
+  title: string;
+  subtitle?: string;
+  description?: string;
+  logo?: ReactNode;
```

Comment:
they must be all ReactNode. You can't know that later you will need to put a tooltip or something else in the title. So better anticipate and accept ReactNode for title, subtitle and description